tests/integration/schema/updates/add/field/kind/invalid_test.go

```diff
@@ -64,30 +64,6 @@ func TestSchemaUpdatesAddFieldKind9(t *testing.T) {
 	testUtils.ExecuteTestCase(t, test)
 }
 
-func TestSchemaUpdatesAddFieldKind13(t *testing.T) {
```

Comment:
todo: Please add a tests in `.../schema/updates/add/field/kind/foo.go` that tests that this field works with PatchSchema